css-positioning/fixed.css

```diff
@@ -0,0 +1,10 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  left: 200px;
```

Comment:
Instead of this big `left` movement, you should set the `width` of the element to `100%` and use `text-align` to center the text 🤓